cmd/podman/system/df.go

```diff
@@ -145,6 +146,7 @@ func printSummary(cmd *cobra.Command, reports *entities.SystemDfReport) error {
 
 	rpt := report.New(os.Stdout, cmd.Name())
 	defer rpt.Flush()
+	report.DefaultFuncs["json"] = rptJSON
```

Comment:
This looks like bad code style, we should not modify the global variable in a different package like that. It will work here because we exit after that but if we start adding another report template after that it will cause bugs.

I will create a PR with better fix